app/controllers/projects_controller.rb

```diff
@@ -70,9 +70,14 @@ def create
   end
 
   def destroy
-    @project.destroy
     respond_to do |format|
-      format.html { redirect_to projects_path, notice: "Project was successfully destroyed." }
+      if @project.title.strip.eql?(params.dig(:project, :title)&.strip)
```

Comment:
Hey @JuanVqz , Out of curiosity, why are we stripping here? should it not be exact match?

Reply:
I noticed that sometimes when editing the project title, it left a space at the end of the title. it might be confusing if you don't realize that when deleting a project. 
Of course, we could normalize the title when saving it (strip it) and remove it here.

what do you think?

Reply:
I do not have a strong reason as to why choose 1 approach over the other. So I would say, lets keep what you have already implemented. Thanks for clarifying.